presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -327,13 +327,13 @@ private void invokeNoMoreSplitsIfNecessary()
         if (partition.getPartition().isPresent()) {
             Optional<HiveBucketProperty> partitionBucketProperty = partition.getPartition().get().getStorage().getBucketProperty();
             if (tableBucketInfo.isPresent() && partitionBucketProperty.isPresent()) {
-                int readBucketCount = tableBucketInfo.get().getReadBucketCount();
+                int tableBucketCount = tableBucketInfo.get().getTableBucketCount();
```

Comment:
Nice revert to https://github.com/prestodb/presto/pull/11749/commits/c223aece743a391fae577a555d438422aa1e3367#diff-20dee960e1c124aae6c511152416a860R317 :P